tools/ui-components/src/button.css

```diff
@@ -3,21 +3,11 @@
 .storybook-button {
   font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
   font-weight: 700;
-  border: 0;
-  border-radius: 3em;
   cursor: pointer;
   display: inline-block;
   line-height: 1;
 }
-.storybook-button--primary {
```

Comment:
The aim is to completely get rid of custom css as much as possible. Here I cleaned what was relevant to the theme support for now.